protocol/src/reqres.rs

```diff
@@ -25,11 +25,16 @@ use libp2p::request_response::RequestResponseCodec;
 use std::io;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Req(pub Vec<u8>);
+pub enum Req {
+    Ping (),
```

Comment:
Please use Rust Style recommendation for spaces: https://doc.rust-lang.org/1.0.0/style/style/whitespace.html
Also, for empty structs no need to put `()`.

Reply:
As I understand the same for Pong ()